### PR TITLE
Text editor settings config

### PR DESCRIPTION
**Pull request by Espe-on**

### CODE REVIEW

**Comment by Espe-on** [comment]
HELL YEAH NO MORE LINTER ERRORS

**Comment by Espe-on** [comment]
Turns out that this breaks ALL autocomplete settings for the editor, am working on getting it working

**Comment by Espe-on** [comment]
Turns out that this breaks ALL autocomplete settings for the editor, am working on getting it working

File: pages/texteditor.tsx [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
 const TextEditor: NextPage = () => <div>
-    <h1>This is the Text Editor</h1>
-    <TextEditorContainer/>
+    <h1>These are the different Text Editors</h1>
+    <h2>A Text Editor with no highlighting or autocomplete</h2>
+    <TextEditorContainer containerClassName="textEditor"
+                         theme="dark"
+                         height="90vh"
+                         width="90vw"
+                         language="unset"
```
Comment (MichaelWalker) [inline pages/texteditor.tsx:15]: why is the language unset here?

File: Models/SupportedLanguages.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿export type SupportedLanguages = 'javascript' | 'typescript' | 'unset';
```
Comment (MichaelWalker) [inline Models/SupportedLanguages.ts:1]: are we planning to actually support typescript?

File: pages/texteditor.tsx (hunk 1 above)
Comment (Espe-on) [reply pages/texteditor.tsx:15]: basically it is the only way to get the text editor to stop giving hints on the code

File: Models/SupportedLanguages.ts (hunk 2 above)
Comment (Espe-on) [reply Models/SupportedLanguages.ts:1]: Ahh sorry I think that is left in from when I was debugging

**Pull request merged**